### PR TITLE
Add filter for post types

### PR DESCRIPTION
**Pull request by edmundcwm**
Add filter for excluding specific post types from search.

### CODE REVIEW

**Comment by rheinardkorf** [comment]
@edmundcwm , it would be ideal to not have the posts get indexed in the first place.  Apply a filter like `skipped_posts` here: https://github.com/xwp/multisite-search/blob/master/php/Admin/class-index.php#L32

The root cause of the problem is here: https://github.com/xwp/multisite-search/blob/master/php/Hooks/class-post-type.php#L44

Even though it defaults to `page` and `post`, this function will index ANYTHING.

**Comment by rheinardkorf** [comment]
A seperate issue, the code in https://github.com/xwp/multisite-search/blob/master/php/CLI/class-index.php#L30 will ONLY index `post` and `page`.  This would need to be fixed to include other types. 

Recommend this as a seperate issue.

File: php/Utility/class-search.php [hunk 1]
```diff
@@ -61,6 +63,9 @@ public function query(
 		$caps     = $escalated_caps ? '(.*)' : $caps;
 		$keywords = str_replace( '**', ' ', $keywords );
 
+		// List of post types to exclude from search.
```
Comment (rheinardkorf) [inline php/Utility/class-search.php]: See other comments on the PR to avoid requiring this code in the first place.

**Review by rheinardkorf — CHANGES_REQUESTED** [review]

**Comment by edmundcwm** [comment]
PR edited!

**Pull request merged**